src/makeandship/elasticsearch/post-document-builder.php

```diff
@@ -389,4 +393,14 @@ public function get_mapping_type($post)
     {
         return Constants::DEFAULT_MAPPING_TYPE;
     }
+
+    /**
+     * Check if an attachment is orphaned media
+     */
+    public static function is_orphaned_media($attachment) 
+    {
+        $post = get_post($attachment->post_parent);
+        $live = $post && ($post->post_status === 'publish' || $post->post_status === 'private');
```

Comment:
Would be good to apply a filter here and allow a project to override/extend this behaviour